backend/sse/build.gradle

```
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '3.5.3'
+    id 'io.spring.dependency-management' version '1.1.7'
+}
+
+group = 'com.bottari'
+version = '0.0.1-SNAPSHOT'
+description = 'sse'
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+repositories {
+    mavenCentral()
+}
```

```suggestion
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(21)
    }
}

// 이거랑 아래꺼 추가
bootJar {
    enabled = true
}

jar {
    enabled = false
}

repositories {
    mavenCentral()
}
```

- Dockerfile 옮겨올 때 이것도 추가해줘
- [관련 댓글](https://github.com/woowacourse-teams/2025-bottari/pull/587#discussion_r2431168583)